enumerables.rb

```diff
@@ -0,0 +1,109 @@
+# Implementing  ruby enum methods
+module Enumerable
+  def my_each
```

Comment:
- [ ]  **#my_each when a block is given when self is a range returns the range itself after calling the given block once for each element in self. The following code should return the range itself**
```
range.my_each(&block)
```